SDKGenerator.njsproj

```diff
@@ -506,7 +506,7 @@
     <Content Include="targets\PythonSdk\source\playfab\PlayFabHTTP.py" />
     <Content Include="targets\PythonSdk\source\playfab\PlayFabManualTest.py" />
     <Content Include="targets\PythonSdk\source\playfab\PlayFabSettings.py" />
-    <Content Include="targets\PythonSdk\source\playfab\__init__.py" />
+    <Content Include="targets\PythonSdk\templates\__init__.py.ejs" />
```

Comment:
You don't need to move this into the templates folder.
You can just rename it in-place to append ".ejs" on the end, and it automatically gets templatized where it is.